index.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="./css/style.css" />
+  </head>
+  <body>
+    <header>
+      <nav class="top-nav">
+        <a href="#" class="logo-txt"> Makoji David </a>
+        <div class="menu">
+          <img src="./img/icons/hamburger.png" alt="logo-icon" class="logo-icon" />
```

Comment:
- [ ] [OPTIONAL] Please use the exact icon from the Figma design. For your information, you can export SVG icons from the Figma design.